config/v1/types_feature.go

```diff
@@ -100,6 +100,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 			"ExperimentalCriticalPodAnnotation", // sig-pod, sjenning
 			"RotateKubeletServerCertificate",    // sig-pod, sjenning
 			"SupportPodPidsLimit",               // sig-pod, sjenning
+			"TLSSecurityProfile",                // sig-network, danehans
```

Comment:
If this gate is off, what happens?  Do you filter out the fields from being set in the API?

Reply:
@deads2k can you help me with this question?